app/views/moderation/images/index.html.haml

```diff
@@ -1,6 +1,10 @@
 %h1 Images externes les plus récentes
 
-- @images.each do |image|
-  .image
-    = image.to_html.html_safe
-    = button_to "Bloquer", moderation_image_path(image.encoded_link), method: :delete, data: { confirm: "Confirmez‑vous vouloir bloquer cette image ?" }
+- if @images.any?
+  %p Images externes les plus récentes
+  - @images.each do |image|
+    .image
+      = image.to_html.html_safe
+      = button_to "Bloquer", moderation_image_path(image.encoded_link), method: :delete, data: { confirm: "Confirmez‑vous vouloir bloquer cette image ?" }
```

Comment:
Maybe we should display the "Bloquer" button only if the image is not blocked ?

When it is already blocked, we could either display a status like "⛔ Bloqué" or a button like "Débloquer" to reverse the allow back the image.

Reply:
Done